tests/TestCase/Error/DebuggerTest.php

```diff
@@ -105,6 +109,7 @@ public function testExcerpt()
         $this->assertTrue(is_array($result));
         $this->assertCount(4, $result);
 
+        $this->skipIf(defined('HHVM_VERSION'), 'HHVM does not highlight php code');
```

Comment:
Using a js highlighter would be a good alternative, but will not work when the developer has no network. Having highlighting degarde gracefully should be ok